styles/work-popup.css

```diff
@@ -0,0 +1,223 @@
+
+.modal {
+  background-color: white;
+  position: fixed;
+  top: 5vh;
+  left: 5vw;
+  width: 90vw;
+  height: 90vh;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  border-radius: 10px;
+  filter: blur(0);
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: row;
+}
+
+.modal-content {
+  justify-content: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(10px);
+  width: 85%;
+  margin: 0 auto;
+}
+
+.modal-head {
```

Comment:
- [ ] When we click the `See project` button in small screens, the bottom and header of the popup box aren't visible, hence navigating through the page is difficult, would you address this issue?

Kindly watch the next screencast to understand more about the context of what I am talking about

https://user-images.githubusercontent.com/56886064/151597156-3bec2366-d30b-4b7f-a35e-fc5ebd6cf5a3.mp4